### ISSUE TITLE
minLimit does not stick to the value when using Range Slider

### ISSUE THREAD

**Issue by sorabh2008** [angular-slider/ng5-slider#76]
Hi

I am trying to use minLimit with the range slider. 
Issue: I am able to slide the pointer at "min" value.

As I am using Steps array, I have tried binding minLimit it to the index as well as value. 

Thanks,

**Comment by piotrdz** [comment]
Yes, it is possible to add this feature. I'll see if I can do it in next release.

**Comment by pjdabrowski** [comment]
Hi,

will it include an option to make minValue or maxValue disabled/read only ?
Is there any chance for some sort of ETA for next release ?
 
Thanks

**Comment by piotrdz** [comment]
@pjdabrowski: It depends on what you mean by that?

For this feature, I think of making minLimit and maxLimit available separately for each slider handle. So there would be a total of six options:
 - `minLimit` and `maxLimit` for both of the slider handles,
 - `minLimitLow` and `maxLimitLow` for low slider handle,
 - `minLimitHigh` and `maxLimitHigh` for high slider handle.

As a special case, if you set `minLimitLow = minLimitHigh`, then the low slider handle will be forced to remain at a single value. The same would apply for the high slider handle.

That should cover the requirements of this feature as well as #83.

@pjdabrowski Does that work for you?

As for the ETA of next release, it will be most likely early May.

**Comment by pjdabrowski** [comment]
@piotrdz Yes - described special case is exactly what i need. Thank you for eta.

**Comment by piotrdz** [comment]
@pjdabrowski  Looks like my ETA was overly optimistic. I don't get enough time to work on this project unfortunately. I will do a 1.2.0 release first with a few other things and then go back to this.

**Comment by pjdabrowski** [comment]
@piotrdz Ok, thank you for the update.

**Comment by Daszczu** [comment]
@piotrdz any updates regarding this? I'd love to have a few initial values disabled from being able to be set

**Comment by joffarex** [comment]
> 
> 
> @pjdabrowski: It depends on what you mean by that?
> 
> For this feature, I think of making minLimit and maxLimit available separately for each slider handle. So there would be a total of six options:
> 
>     * `minLimit` and `maxLimit` for both of the slider handles,
> 
>     * `minLimitLow` and `maxLimitLow` for low slider handle,
> 
>     * `minLimitHigh` and `maxLimitHigh` for high slider handle.
> 
> 
> As a special case, if you set `minLimitLow = minLimitHigh`, then the low slider handle will be forced to remain at a single value. The same would apply for the high slider handle.
> 
> That should cover the requirements of this feature as well as #83.
> 
> @pjdabrowski Does that work for you?
> 
> As for the ETA of next release, it will be most likely early May.

@piotrdz did you not implement  `minLimitLow` and `maxLimitLow` for low slider handle,`minLimitHigh` and `maxLimitHigh` for high slider handle? I am asking this because I could not find those options in `Option` and I really would like to have this feature rn.

**Comment by pjdabrowski** [comment]
If this helps here is my workaround for disabling one slider:
```

  @ViewChild('slider', {static: true}) slider: any;

  ngAfterViewInit(): void {
    this.disableUpperBoundSlider(this.slider);
  }

  private disableUpperBoundSlider(slider: any) {
    slider.getPointerElement(1).off('mousedown');
    slider.getPointerElement(1).off('touchstart');
    slider.getPointerElement(1).off('focus');
    slider.getNearestHandle = () => 0;
  }
``` 

I imagine you can limit its range in similar way. GL !

**Comment by joffarex** [comment]
@pjdabrowski I dont get it. where does this code snipper fit? do you have extended the slider?

**Comment by pjdabrowski** [comment]
@joffarex  it goes into your component that uses ng5-slider. And it is a hack obviously ;)
 I updated my previous example.

**Comment by BrentLeemans** [comment]
Hi @piotrdz any update on this feature? I would love to see this getting implemented.
